Drivers/Src/LTC6811.c

```diff
@@ -92,13 +114,17 @@ void LTC6811_Init(cell_asic *battMod){
     mutexExists = true;
   }
 
-	BSP_SPI_Init();				// Initialize SPI1 for voltage board	
-	
+  spi_os.pend = LTC6811_Pend;
+  spi_os.post = LTC6811_Post;
+	BSP_SPI_Init(&spi_os);				// Initialize SPI1 for voltage board	
```

Comment:
In the actual BSP_Init function, you should load these function memory addresses to something in the library. Then whenever you access Minions, you would call those functions and they would run the pend and post. Also, what is the variable that this semaphore is being used on? I don't see one in BSP_SPI.c.